packages/stark-testing/karma.conf.typescript.ci.js

```diff
@@ -16,9 +16,9 @@ const rawKarmaConfig = {
 
 	// list of files / patterns to load in the browser
 	files: [
-		{pattern: helpers.root(helpers.getAngularCliAppConfig().test)},
-		{pattern: helpers.root("src/**/*.ts")},
-		{pattern: helpers.root("src/**/*.html")}
+		{ pattern: helpers.root(helpers.getAngularCliAppConfig()["architect"]["test"]["options"].main) },
```

Comment:
Again, I think we should just use the dot syntax whenever possible...

Reply:
done